run_project_tests.py

```diff
@@ -996,6 +996,7 @@ def __init__(self, category: str, subdir: str, skip: bool = False, stdout_mandat
         TestCategory('frameworks', 'frameworks'),
         TestCategory('nasm', 'nasm'),
         TestCategory('wasm', 'wasm', shutil.which('emcc') is None or backend is not Backend.ninja),
+        TestCategory('zig', 'zig', shutil.which('zig') is None)
```

Comment:
You also need to check that the backend is ninja here, no zig for visual studio or xcode